### PR TITLE
Make PWA ready

### PR DESCRIPTION
**Pull request by unasuke**
## Why?
For push notifications (especially for iOS)

## ref
https://web.dev/learn/pwa/web-app-manifest?hl=ja

## iOS
image

image

## Android
image

image

image

### CODE REVIEW

File: app/views/pwa/manifest.json.jb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+manifest = {
+  name: "Kaigi on Rails 2023 conference-app",
+  short_name: "Kaigi on Rails 2023",
```
Comment (unasuke) [inline app/views/pwa/manifest.json.jb:3]: Too long? How do you think @okuramasafumi

Comment (okuramasafumi) [reply app/views/pwa/manifest.json.jb:3]: "Kaigi on Rails" seems enough when icon shows it's 2023.

File: app/views/pwa/manifest.json.jb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      src: image_path("icons/2023/1024_maskable.png"),
+      type: "image/png",
+      sizes: "1024x1024",
+      purpose: "maskable",
+    },
+  ],
+}
+
+manifest
```
Comment (okuramasafumi) [inline app/views/pwa/manifest.json.jb:30]: Do we need this line instead of just returning a literal?

Comment (unasuke) [reply app/views/pwa/manifest.json.jb:30]: tashikani

File: app/views/pwa/manifest.json.jb (hunk 1 above)
Comment (unasuke) [reply app/views/pwa/manifest.json.jb]: 342d67f

File: app/views/pwa/manifest.json.jb (hunk 2 above)
Comment (unasuke) [reply app/views/pwa/manifest.json.jb]: 9d3d3cf

File: app/views/pwa/manifest.json.jb (hunk 1 above)
Comment (unasuke) [reply app/views/pwa/manifest.json.jb]: | iOS | Android |
| --- | --- |
| image | image |

looks good.

File: Gemfile [hunk 3]
```diff
@@ -62,6 +62,7 @@ gem "woothee"
 gem "sentry-rails"
 gem "scout_apm"
 
+gem "jb"
```
Comment (okuramasafumi) [inline Gemfile:65]: We're using `jb` only for `image_path` which can be replaced with simple strings?

Comment (unasuke) [reply Gemfile:65]: Possible.

Comment (unasuke) [reply Gemfile]: 0189854b538f

**Review by okuramasafumi — APPROVED** [review]
Thanks!

**Pull request merged**